src/index.html

```diff
@@ -2,16 +2,186 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
```

Comment:
Still 3rd item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) isn't fixed
```suggestion
    <meta 
    	name="viewport"
```